pyrasterframes/src/main/python/docs/nodata-handling.pymd

```diff
@@ -135,14 +139,15 @@ Now we will use the @ref:[`rf_mask_by_value`](reference.md#rf-mask-by-value) to
 ```python, mask_blu
 with_nd = rf_convert_cell_type('blue', masked_blue_ct)
 masked = one_mask.withColumn('blue_masked',
-                             rf_mask_by_value(with_nd, 'mask', lit(1))) \
-                 .drop('nodata', 'defect', 'cloud8', 'cloud9', 'cirrus', 'blue')
+    rf_mask_by_value(with_nd, 'mask', lit(1))) \
+    .drop('nodata', 'defect', 'cloud8', 'cloud9', 'cirrus', 'blue')
 ```
 
 We can verify that the number of NoData cells in the resulting `blue_masked` column matches the total of the boolean `mask` _tile_ to ensure our logic is correct.
 
-```python, show_masked, results='raw'
-masked.select(rf_no_data_cells('blue_masked'), rf_tile_sum('mask')).show(10)
+```python, show_masked
+counts = masked.select(rf_no_data_cells('blue_masked'), rf_tile_sum('mask'))
+display(counts)
```

Comment:
In some of these examples if we can just have a spark DF on the last line that will have `display` evaluated on  it, it would improve readability some to revert back.

Reply:
I like that idea... not sure I realized that behavior at the time.